frontend/src/assets/primevue.scss

```diff
@@ -29,3 +58,55 @@
     margin-left: 1rem;
   }
 }
+
+// Primary color overrides for buttons and action items (checkboxes etc)
+// Note this could be eventually replaced by a custom themeing (which has JUST been introduced in Primevue)
+// once it is more settled implementation-wise
```

Comment:
Wouldn't adopt the PrimeVue open-source theme customization yet, just came out and has all these notes about changes. This is easy enough at this point